packages/shared/src/components/icons/Arrow/Straight/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { ReactElement } from 'react';
+import Icon, { IconProps } from '../../../Icon';
+import FilledIcon from './filled.svg';
+
+const StraightArrowIcon = (props: IconProps): ReactElement => (
+  <Icon {...props} IconPrimary={FilledIcon} IconSecondary={FilledIcon} />
```

Comment:
Is IconSecondary required since it's the same icon?

Reply:
When we don't have a counterpart, we fill it with the same value.